store/v2/root/store.go

```diff
@@ -340,42 +331,10 @@ func (s *Store) loadVersion(v uint64, upgrades *corestore.StoreUpgrades) error {
 	return nil
 }
 
-func (s *Store) SetCommitHeader(h *coreheader.Info) {
-	s.commitHeader = h
-}
-
 // WorkingHash writes the changeset to SC and SS and returns the workingHash
 // of the CommitInfo.
 func (s *Store) WorkingHash(cs *corestore.Changeset) ([]byte, error) {
```

Comment:
should we remove this?

Reply:
it is unused, i don't see a use case for it, but maybe I'm wrong?

Reply:
oh was looking at the wrong diff, yea meant to remove this func